CHANGELOG.md

```diff
@@ -0,0 +1,40 @@
+# 0.3.0
+
+This version roughly corresponds with all the changes made to the Temporal polyfill as of the November TC39 Plenary meeting.
```

Comment:
```suggestion
This version roughly corresponds with all the changes made to the Temporal polyfill as of the November 2021 TC39 Plenary meeting.
```

Reply:
There wasn't a November meeting - I think this should be October :smile:

Reply:
Oops - Fixed :D